### DISCUSSION TITLE
关于程序窗口问题的反馈

### DISCUSSION THREAD

**Discussion by Jerry95000** [xiaoyifang/goldendict-ng#1188] [Q&A]
[当我使用win11系统自带的Copilot或Onenote中的停靠到桌面功能的时候，如果goldendict程序（最新版）正在运行，有时候词典主界面的窗口会自动弹出来，点击右上方的叉号也无法关闭窗口（如图），并且这时在这个窗口里无法进行任何操作。只能通过右击右下方的小图标关闭并重启程序解决。
另外，还有个小问题。手动在主界面输入单词，并按回车键时，有时候会出现没有任何反应的情况（手动输入的内容没有拼写错误），这时需要用鼠标去点击弹出的下拉框中出现的词条才能进行查询。
<img width="1280" alt="Snipaste_2023-10-02_17-04-55" src="https://github.com/xiaoyifang/goldendict-ng/assets/117874864/a37a7af1-d412-40b4-a259-cca154d26cfb">

**Comment by xiaoyifang** [comment]
which release ,the version in the Help->About

**Reply by Jerry95000** [reply]
Goldendict-ng 23.09.29-MoonCake.20230930.7f0f8778
Qt 6.5.2 Visual C++ Compiler 192930152 windows winnt 10.0.22631 x86_64-little_endian-llp64
Flags:USE_XAPIAN MAKE_ZIM_SUPPORT USE_ICONV MAKE_CHINESE_CONVERSION_SUPPORT no_ffmpeg_player
